.travis.yml

```diff
@@ -88,7 +89,28 @@ matrix:
         apt:
           packages:
             - libspatialindex-dev
+    - env: TOXENV=py38-anaconda
+      name: "Python3.8 (Linux + Anaconda)"
+      python: 3.8
+      before_install:
+          - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+          - bash miniconda.sh -b -p $HOME/miniconda
+          - export PATH="$HOME/miniconda/bin:$PATH"
+          - hash -r
+          - conda config --set always_yes yes --set changeps1 no
+          - conda install setuptools
+          - conda update -q conda
+          - conda create -n xclim -c conda-forge python=$TRAVIS_PYTHON_VERSION
+          - conda env update -f environment.yml
+          - conda install pytest coveralls pytest-cov
```

Comment:
This installs pytest in the base environment. I think we should call `conda activate xclim` earlier!